lib/src/ast.dart

```diff
@@ -12,16 +12,23 @@ abstract class Query {
 }
 
 /// Text query to match [text].
-///
-/// [isExactMatch] is set when the [text] was inside quotes.
 class TextQuery implements Query {
   final String text;
-  final bool isExactMatch;
```

Comment:
This is a breaking change. As `PhraseQuery` already extends TextQuery, wouldn't it be possible to keep this one, and set it to `false` for non-`PhraseQuery`, and `true` for `PhraseQuery`?

It would be nice if we could keep `text` too, as close to preserve the original text as possible.

Reply:
Ok. Let me work on later today. I was thinking about something similar but didn't want to have redundancy in the code. But what you suggested makes sense.

Reply:
Done